prestoadmin/slider/server.py

```diff
@@ -67,3 +74,77 @@ def slider_uninstall():
     Uninstall slider from the slider master.
     """
     sudo('rm -r "%s"' % (env.conf[DIR]))
+
+
+def get_slider_bin(conf):
+    return os.path.join(conf[DIR], 'bin', 'slider')
+
+
+def run_slider(slider_command, conf):
+    with shell_env(JAVA_HOME=conf[JAVA_HOME],
+                   HADOOP_CONF_DIR=conf[HADOOP_CONF]):
+        return sudo(slider_command, user=conf[SLIDER_USER])
+
+
+@task
+@requires_conf
+@task_by_rolename(SLIDER_MASTER)
+def install(presto_yarn_package):
+    """
+    Install the presto-yarn package on the cluster using slider. The
+    presto-yarn package takes the form of a zip file that conforms to slider's
+    packaging requirements. After installing the presto-yarn package the presto
+    application is registered with slider.
+
+    The name of the presto application is arbitrary and set in the slider
+    configuration file. The default is PRESTO
+
+    :param presto_yarn_package: The zip file containing the presto-yarn
+    package as structured for slider.
+    """
+    conf = env.conf
+    package_filename = os.path.basename(presto_yarn_package)
+    package_file = os.path.join('/tmp', package_filename)
+
+    result = put(presto_yarn_package, package_file)
+    if result.failed:
+        abort('Failed to send slider application package to %s on host %s' %
+              (package_file, env.host))
+
+    slider_command = '%s package --install --package %s --name %s' % \
+                     (get_slider_bin(conf), package_file, conf[APPNAME])
+
+    try:
+        run_slider(slider_command, conf)
+
+        env.conf[PRESTO_PACKAGE] = package_filename
```

Comment:
We need this to fill in the appConfig.json for server start. No use yet, but that's the next thing on the list.